content/en/downloads/_index.md

```diff
@@ -9,12 +9,12 @@ images: []
 weight: 100
 layout: default
 ---
-The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [maven central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.1.Final](https://github.com/jbosstm/narayana/tree/7.2.1.Final)) or download it from github tag (i.e [7.2.1.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.1.Final))
+The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [maven central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
 
 
 | | Version | Date | License |
 | ---------------------------------------- | ----------- | ----------------- | ---------------- |
-| [Narayana Source Code](https://github.com/jbosstm/narayana/releases/tag/7.2.1.Final) | 7.2.1.Final | 2025-01-31 | Apache-2.0
+| [Narayana Source Code](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final) | 7.2.2.Final | 2025-01-31 | Apache-2.0
```

Comment:
I think the date on here is not what I would expected, please can I ask you to take a look @jmfinelli ?

Reply:
Thanks for spotting this, Tom 😄